docs/getting-started.md

```diff
@@ -105,3 +110,35 @@ Or to start the auto-restarting development server:
 ```sh
 make start-dev
 ```
+
+Run Docker for development
```

Comment:
```suggestion
Running with docker-compose
```

Reply:
Suggesting this because I think docker-compose isn't strictly for dev environments.

Reply:
Ignore this. I think docker compose in prod probably looks a bit different anyway.